README.md

```diff
@@ -271,15 +271,24 @@ Open each `text.po` domain file and, for each `msgid` string, add a correspondin
 ```sh
 i18n_tool generate
 ```
-Before running the `i18n_tool extract` or `i18n_tool generate` commands, appropriate settings for xblock-poll need to be added to `edx-platform/conf/locale/config.yaml`.
-You should have to mention `poll` as a third_party app and `poll.po` in `django.po` list under `generate_merge` section.
+Before running `i18n_tool extract` or `i18n_tool generate` as described above, appropriate settings for xblock-poll need to be added to `edx-platform/conf/locale/config.yaml`. In particular, you'll need to add `poll` as a `third_party` app and list `poll.po` under `django.po` in the `generate_merge` section:
 
-You can copy the files into xblock-poll repo via using unix `cp` command
-* 'cd <path to edx-platform repo>/'
-* 'cp conf/locale/en/LC_MESSAGES/poll.po <Path to Xblock_Poll repo>/poll/translations/en/LC_MESSAGES/text.po'
-* 'cp conf/locale/en/LC_MESSAGES/poll.mo <Path to Xblock_Poll repo>/poll/translations/eo/LC_MESSAGES/text.mo'
+...
+   third_party:
+       - ...
+       - poll
+   ...
+   generate_merge:
+       django.po:
+           - ...
+           - poll.po
```

Comment:
@asadiqbal08 This doesn't render correctly:

![broken-markdown](https://cloud.githubusercontent.com/assets/961441/16147594/59cf5036-3484-11e6-8937-d597d43ae1ee.png)

Each of the lines in this example (L276-L284) needs to start with four spaces (currently it's only three).